oak-core/src/main/java/org/apache/jackrabbit/oak/namepath/impl/GlobalNameMapper.java

```diff
@@ -67,10 +67,15 @@ protected static boolean isHiddenName(String name) {
     protected static boolean isExpandedName(String name) {
         if (name.startsWith("{")) {
             int brace = name.indexOf('}', 1);
-            return brace != -1 && name.substring(1, brace).indexOf(':') != -1;
-        } else {
-            return false;
+            if (brace != -1) {
+                String namespace = name.substring(1, brace);
+                // the empty namespace is valid as well, otherwise it always contains a colon (as it is a URI)
```

Comment:
empty namespace used in expanded form was not detected correctly @reschke

Reply:
If there's a bug here, we should fix it independently; we may also want to backport that fix to 1.22.

Could you please open a Jira issue? Do we lack test coverage?

Reply:
Also, we need to check that the "internal" namespace is handled consistently throughout and has sufficient test coverage.

Reply:
Done in https://issues.apache.org/jira/browse/OAK-11450, yes seems test coverage is not complete here.